manifests/bird_exporter.pp

```diff
@@ -63,7 +63,7 @@
 # @author Tim Meusel <tim@bastelfreak.de>
 #
 class prometheus::bird_exporter (
-  String $download_extension              = '',
+  Optional[String[1]] $download_extension = undef,
```

Comment:
It's important to still support empty strings, that's how the defined resource `prometheus::daemon` knows to just download plain file which I make a lot of use of when archives aren't available.